### PR TITLE
Refactor market simulation to be more readable and set the start conf…

### PR DESCRIPTION
**Pull request by tomschw**
…iguration to be computable in less than 1 minute.

### CODE REVIEW

**Comment by Bouncner** [comment]
Have you tested the whole pipeline?

I wont be able to test it until next week. But I would trust you and merge when you say you tested the whole pipeline. ;)

**Comment by tomschw** [comment]
Except for a small error in the monte_carlo file, everything seems to work as expected and the graphs don't seem abnormal.
One current problem is, that the demand_learning notebook wants to load the S1 notebooks, which currently aren't getting created. If we keep it like that, it would probably be wise to simply run all notebooks with every strategy, simply so we can expect them. I found an easy way to do, which can be found in issue #8 .

**Comment by Bouncner** [comment]
Then please change demand_learning to use the scenario that we are currently creating as the default.

Which bug do you mean?

**Comment by tomschw** [comment]
I already use the current scenario as default but in demand_learning we compare the two scenarios at the end. Shall I comment it out for the moment?

The problem was that I didn't remove all unnecessary output, which was only used when Rainer still calculated the logit in the notebook. I will commit the changed notebook at the end of today.

**Comment by Bouncner** [comment]
I see your point.

Let’s ignore that issue for now, as I think it’s clear that users have to generate both conpare scenarios.

I’ll review after your fix.

**Comment by tomschw** [comment]
fixed

File: monte_carlo.ipynb [hunk 1]
```diff
@@ -174,7 +174,7 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 35,
```
Comment (Bouncner) [inline monte_carlo.ipynb]: Please commit a cleared or ran-once notebook and I’ll merge.

**Review by Bouncner — CHANGES_REQUESTED** [review]

File: monte_carlo.ipynb (hunk 1 above)
Comment (tomschw) [reply monte_carlo.ipynb]: Done

**Pull request merged**